### PR TITLE
Final touches

### PR DESCRIPTION
**Pull request by Rishi-Mishra0704**
# Production

**_Hello respected reviewer , in this PR , I created a budget app and deployed it on render . It has many amazing features such as_**

- Login and Signup pages
- Can add an expense 
- Can add a category
- Link expense to category
- Add models 
- Add views
- Add controllers

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,165 @@
...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+Please give a ⭐️ if you like this project
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
```
Comment (t-yanick) [inline README.md:155]: - [ ] As per project requirements, the README should contain in the `Acknowledgements` section or similar that gives appropriate credit to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) the [author of the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), as required by the [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/). This is currently missing in your README. kindly add it

File: app/views/splash/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    <div class="splash-text">
+        <p class="lead fs-3">Manage your finances with ease</p>
+        <p class="lead fs-3">Track your spending</p>
+        <p class="lead fs-3">Set goals</p>
+        <p class="lead fs-3 mb-5">Save money</p>
+    </div>
+
+    <div class="log-in-container">
+        <%= link_to "Log in", new_user_session_path, class: "button-splash btn btn-primary log" %>
+        <%= link_to "Sign in", new_user_registration_path, class: "button-splash btn btn-primary log" %>
```
Comment (t-yanick) [inline app/views/splash/index.html.erb]: - [ ] I think the placeholder text you should be Sign up and not Sign in as Log in and Sign in technically mean the same thing

**Review by t-yanick — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED

Hi @Rishi-Mishra0704 , 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Video presents all features of the project
:heavy_check_mark: Video within recommended limit of 5minutes
:heavy_check_mark: The video contains a clear and objective description about the project in professional English
:heavy_check_mark: The video contains a demo of the project, highlighting all the project features.
:heavy_check_mark: Descriptive Readme file with live demo and video presentation links
:heavy_check_mark: Presence of splash screen
:heavy_check_mark: Sign up and log in pages added and work as expected
:heavy_check_mark: Home page and expense page added as per requirements
:heavy_check_mark: Use of Postgresql
:heavy_check_mark: Use of devise for authentication
:heavy_check_mark: Unit and integration tests added

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Rishi-Mishra0704** [comment]
Respected reviewer, I didn't deploy the project again after making the change, What I'm trying to say is the "sign in" will remain the same in deployed website. Please check the code to confirm the change. Thank you so much

**Review by elisha2kyakpo1 — APPROVED** [review]
Hi 👋 @Rishi-Mishra0704, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

👍 You did well adding the owner of the design used in the readme file as requested by the previous reviewer.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**